src/test/kotlin/com/example/fhirvalidator/controller/ValidateControllerTest.kt

```diff
@@ -40,12 +40,23 @@ internal class ValidateControllerTest {
 
     @Test
     fun getResourcesToValidate_returns_list_of_entries_when_passed_searchset() {
+        val childBundle = Bundle()
+        val bundle = Bundle()
+        bundle.type = Bundle.BundleType.SEARCHSET
+        val bundleEntry = bundle.addEntry()
+        bundleEntry.resource = childBundle
+        assertEquals(listOf(childBundle), testValidateController.getResourcesToValidate(bundle))
+    }
+
+    // Split from above test. Only Bundles of Bundles should return a list
```

Comment:
remove

Reply:
Please elaborate?

Reply:
This is to test the exception case (for $release) for searchSet tests.